test/libs/BytecodeStorageV1_BackwardsCompatibleReadsMock.test.ts

```diff
@@ -0,0 +1,380 @@
+import {
+  BN,
+  constants,
+  expectEvent,
+  expectRevert,
+  balance,
+  ether,
+} from "@openzeppelin/test-helpers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { Contract } from "ethers";
+import type { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+
+import {
+  T_Config,
+  getAccounts,
+  deployAndGet,
+  assignDefaultConstants,
+} from "../util/common";
+import {
+  SQUIGGLE_SCRIPT,
+  SKULPTUUR_SCRIPT_APPROX,
+  CONTRACT_SIZE_LIMIT_SCRIPT,
+  GREATER_THAN_CONTRACT_SIZE_LIMIT_SCRIPT,
+  MULTI_BYTE_UTF_EIGHT_SCRIPT,
+} from "../util/example-scripts";
+
```

Comment:
Nit: Not totally necessary but nice for DX, you can create an interface that extends T_Config and includes the expected fields that you add to the config so that you can call contract methods with typesafety/get auto complete while writing tests. See the dependency registry tests for example

Reply:
Not _necessary_, but is goated feedback. Thank you for calling this out 🙏 🔥 

I will update to do so, and resolve this comment once I've pushed an update.

Reply:
Updated!